--- a/src/Concerns/InteractsWithActions.php
+++ b/src/Concerns/InteractsWithActions.php
@@ -336,7 +336,7 @@ trait InteractsWithActions

         if (! $action instanceof Action) {
-            throw new InvalidArgumentException('Actions must be an instance of ' . Action::class . ". The [{$methodName}] method on the Livewire component returned an instance of [" . get_class($action) . '].');
+            throw new InvalidArgumentException('Actions must be an instance of ' . Action::class . ". The [{$methodName}] method on the Livewire component returned an instance of [" . (is_object($action) ? get_class($action) : gettype($action)) . '].');
         }

